bibliopixel/drivers/visualizer.py

```diff
@@ -38,14 +33,15 @@ def __init__(self, num=0, width=0, height=0, pixelSize=15, port=1618, stayTop=Fa
             if "windows" in operating_system:
                 exe_string = "start python"
             elif "darwin" in operating_system:
-                exe_string = "python"
+                exe_string = "python2.7"
                 suffix = "&"
             else:
-                exe_string = "python"
+                exe_string = "python2.7"
```

Comment:
You have no idea.... so, the reason it's like that is because I am using tkinter for the Visualizer UI which is freaking picky about what thread/process the UI runs on and I never got it to run with multiprocessing right. So I decided not to waste more time and just spawned the process. However, I would rather get a new visualizer than rework this one. No one really uses it to my knoweldge. Not even me. I have a secret implementation I build in C#/WPF (so Windows only.. I know, I know) that is an order of magnitude faster. But I would LOVE any suggestions for something better that's NOT in tkinter!